src/main/resources/bidder-config/interactiveoffers.yaml

```diff
@@ -1,14 +1,14 @@
 adapters:
   interactiveoffers:
     enabled: false
-    endpoint: https://prebid-server.ioadx.com/bidRequest/?partnerId=d9e56d418c4825d466ee96c7a31bf1da6b62fa04
+    endpoint: https://prebid-server.ioadx.com/bidRequest/?partnerId=
```

Comment:
use placeholder here like {{PARTNERID}} and use replace while constructing url

Reply:
changed